Add SEO (#98)
OpenLiberty/guide-microprofile-metrics: README.adoc

:page-permalink: /guides/{projectid}
:page-related-guides: ['rest-intro', 'microprofile-health', 'cdi-intro']
:common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
:page-seo-title: Monitoring the metrics of a Java microservice using Eclipse MicroProfile Metrics

> :page-seo-description: A tutorial on how to expose both system and custom application metrics from a Java microservice and monitor endpoints using Eclipse MicroProfile Metrics.

Reviewer: ```suggestion
:page-seo-description: A getting started tutorial and an example on how to expose system and application metrics from a Java microservice and monitor metrics endpoints using Eclipse MicroProfile Metrics.
```

Reviewer: We can consider changing from "a Java microservice" to "Java microservices" too.
